packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -264,17 +313,35 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
   }
 
   @autobind
+  protected onSuggestionRemove(ev: React.MouseEvent<HTMLElement>, item: IPersonaProps, index: number) {
+    if (this.props.onRemoveSuggestion) {
+      this.props.onRemoveSuggestion(item);
+    }
+    this.suggestionStore.removeSuggestion(index);
+    this.setState({ suggestionsVisible: true });
+  }
+
+  @autobind
   protected onInputFocus(ev: React.FocusEvent<HTMLInputElement | BaseAutoFill>) {
     this.selection.setAllSelected(false);
-    if (this.input.value) {
-      this.setState({ suggestionsVisible: true });
+    if (this.input.value === '' && this.props.onInputFocus) {
+      this.onEmptyInputFocus();
+      this.setState({
+        isMostRecentlyUsedVisible: true,
+        moreSuggestionsAvailable: false,
+        suggestionsVisible: true
+      });
```

Comment:
I'm not sure about `isMostRecentlyUsedVisible` as state. What if I want the most recently used to always appear, what if I only want them to appear sometimes.